.github/actions/custom-actions/aselo_development_custom/action.yml

```diff
@@ -28,20 +29,70 @@ runs:
     #   with:
     #     ssm_parameter: "DEV_AS_IWF_API_COUNTRY_CODE"
     #     env_variable_name: "IWF_API_COUNTRY_CODE"
-    # Append environment variables
-    - name: Add IWF_API_USERNAME
-      run: echo "IWF_API_USERNAME=${{ env.IWF_API_USERNAME }}" >> .env
-      shell: bash
-    - name: Add IWF_API_PASSWORD
-      run: echo "IWF_API_PASSWORD=${{ env.IWF_API_PASSWORD }}" >> .env
-      shell: bash
-    - name: Add IWF_API_URL
-      run: echo "IWF_API_URL=${{ env.IWF_API_URL }}" >> .env
-      shell: bash
-    # - name: Add IWF_API_ENVIRONMENT
-    #   run: echo "IWF_API_ENVIRONMENT=${{ env.IWF_API_ENVIRONMENT }}" >> .env
-    #   shell: bash
-    # - name: Add IWF_API_COUNTRY_CODE
-    #   run: echo "IWF_API_COUNTRY_CODE=${{ env.IWF_API_COUNTRY_CODE }}" >> .env
-    #   shell: bash
-    
\ No newline at end of file
+  - name: Set Aselo Twitter Consumer Key
+    uses: "marvinpinto/action-inject-ssm-secrets@latest"
+    with:
+      ssm_parameter: "TWITTER_CONSUMER_KEY"
+      env_variable_name: "TWITTER_CONSUMER_KEY"
+  - name: Set Aselo Twitter Consumer Secret
+    uses: "marvinpinto/action-inject-ssm-secrets@latest"
+    with:
+      ssm_parameter: "TWITTER_CONSUMER_SECRET"
+      env_variable_name: "TWITTER_CONSUMER_SECRET"
+  - name: Set helpline Twitter Access Token
+    uses: "marvinpinto/action-inject-ssm-secrets@latest"
+    with:
+      ssm_parameter: "TWITTER_ACCESS_TOKEN_1398343158954663937_AseloDev"
+      env_variable_name: "TWITTER_ACCESS_TOKEN"
+  - name: Set helpline Twitter Access Secret
+    uses: "marvinpinto/action-inject-ssm-secrets@latest"
+    with:
+      ssm_parameter: "TWITTER_ACCESS_TOKEN_SECRET_1398343158954663937_AseloDev"
+      env_variable_name: "TWITTER_ACCESS_TOKEN_SECRET"
+  - name: Set helpline Twitter Flex Flow SID
+    uses: "marvinpinto/action-inject-ssm-secrets@latest"
+    with:
+      ssm_parameter: "DEV_TWILIO_AS_TWITTER_FLEX_FLOW_SID"
+      env_variable_name: "TWILIO_TWITTER_FLEX_FLOW_SID"
+  - name: Set helpline Facebook Page Access Token
+    uses: "marvinpinto/action-inject-ssm-secrets@latest"
+    with:
+      ssm_parameter: "FACEBOOK_PAGE_ACCESS_TOKEN_105220114492262_TL-Staging"
+      env_variable_name: "FACEBOOK_PAGE_ACCESS_TOKEN"
+  - name: Set helpline Instagram Flex Flow SID
+    uses: "marvinpinto/action-inject-ssm-secrets@latest"
+    with:
+      ssm_parameter: "DEV_TWILIO_AS_INSTAGRAM_FLEX_FLOW_SID"
+      env_variable_name: "TWILIO_INSTAGRAM_FLEX_FLOW_SID"
```

Comment:
Because of the recent changes in https://github.com/techmatters/serverless/pull/191/commits/0e77b6dbe8ebfc714036fc5828003b1441cf47a8, we also need `FACEBOOK_APP_SECRET` from Parameter Store (same name in SSM). Can we add here before merging? (I can push the commit by myself if you want).